src/features/events/components/EventParticipantsList.tsx

```diff
@@ -10,12 +10,14 @@ import { ZetkinEvent } from 'utils/types/zetkin';
 
 interface EventParticipantsListProps {
   data: ZetkinEvent;
+  filterState: string;
```

Comment:
I think this (and all related references to the same string) should be called `filterString`, not `filterState`.